### PR TITLE
Fix bug where asset base URL was not being used again

### PR DESCRIPTION
**Pull request by sgarner**
51748ca introduced a regression which prevented the correct Assetic base URL from being used.

See also my comment at https://github.com/hearsayit/HearsayRequireJSBundle/commit/51748ca46a0f501f72d16b5f0cbe3f7c1e2677b3#commitcomment-5008154

### CODE REVIEW

**Comment by ihortymoshenko** [comment]
@sgarner, thanks for the contribution. This PR breaks tests. Please look at the link https://travis-ci.org/hearsayit/HearsayRequireJSBundle/builds/16566029 for more details.

**Comment by sgarner** [comment]
@IgorTimoshenko Ah... OK, I think I have fixed the tests now.

File: Configuration/ConfigurationBuilder.php [hunk 1]
```diff
@@ -171,18 +171,25 @@ public function setPath($path, $locations)
...
         if ($this->container->getParameter('assetic.use_controller')
             && $this->container->isScopeActive('request')) {
-            return $this->container->get('request')->getBaseUrl();
+            $baseUrl = $this->container->get('request')->getBaseUrl();
         }
 
-        $baseUrl = $this->container->isScopeActive('templating.helper.assets')
-            ? $this->container->get('templating.helper.assets')->getUrl('')
-            : '';
+        if ($assetHelper = $this->container->get('templating.helper.assets', ContainerInterface::NULL_ON_INVALID_REFERENCE)) {
```
Comment (ihortymoshenko) [inline Configuration/ConfigurationBuilder.php]: String length more than 120 characters. Please extract the `$assetHelper` variable from the condition.

File: Configuration/ConfigurationBuilder.php [hunk 2]
```diff
@@ -171,18 +171,25 @@ public function setPath($path, $locations)
...
+            // Remove ?version from the end of the base URL
+            if (($pos = strpos($baseUrl, '?')) !== false) {
+                $baseUrl = substr($baseUrl, 0, $pos);
+            }
+        }
 
-        // Remove ?version from the end of the base URL
-        if (($pos = strpos($baseUrl, '?')) !== false) {
-            $baseUrl = substr($baseUrl, 0, $pos);
+        // Remove trailing slash, if there is one
```
Comment (ihortymoshenko) [inline Configuration/ConfigurationBuilder.php:191]: Why not just use the `rtrim` function?

``` php
// ...
return \rtrim($baseUrl, '/');
// ...
```

Comment (sgarner) [inline Configuration/ConfigurationBuilder.php:191]: I normally reserve rtrim() for removing whitespace, but yes that is an easier way to do it!

I have added a commit with this and the line length change.

**Comment by ihortymoshenko** [comment]
@sgarner, great! Thanks!

**Pull request merged**